client/main.lua

```diff
@@ -341,6 +343,7 @@ local function createZones(garageName, garage, accessPoint, accessPointIndex)
             end,
             debug = config.debugPoly,
         })
+        table.insert(allZones, {spherePark = parkZone, garageName = garageName, garage = garage, accessPoint = accessPoint, accessPointIndex = accessPointIndex})
```

Comment:
```suggestion
        allZones[#allZones + 1] = {spherePark = parkZone, garageName = garageName, garage = garage, accessPoint = accessPoint, accessPointIndex = accessPointIndex}
```

avoid using `table.insert()`